style.css

```diff
@@ -396,17 +600,616 @@ ul {
   justify-content: space-around;
 }
 
-.lists h4 {
+#about-myself .about-2 .lists h4 {
   margin-right: 0;
   align-items: center;
   justify-content: flex-start;
-  max-width: 60vw;
-  padding: auto;
+  max-width: 60vw;/* padding: auto; */
```

Comment:
- [ ] It's a good thing you currently load this part of your portfolio dynamically. How about removing the whole of this completely? Commenting old pieces of code in a workspace makes the code looks kinda messy :hammer_and_wrench: